style.css

```diff
@@ -1067,13 +1075,105 @@ ul {
   list-style-type: none;
 }
 
+/* Form */
+
 .form-container {
-  border: 3px solid red;
+  border: 1px solid transparent;
   background: var(--main-purple-color);
   border-radius: 100px 0 0 0;
   background-image: url(assets/contact-form-shapes-mobile.svg);
```

Comment:
Kindly update the positioning of the background image in your contact form to better match the figma design🙏

Your design:
<img width="208" alt="image" src="https://user-images.githubusercontent.com/37116322/228344925-23105d82-7d9c-4fba-bc21-10f3541203db.png">

Figma design:
<img width="142" alt="image" src="https://user-images.githubusercontent.com/37116322/228345007-fd93a712-3c3c-44b8-ac50-4220ec700c43.png">

A few things to note:
- The shapes under the textarea are not showing

Reply:
resolved the Issue.